drivers/sensor/voltage/Kconfig

```diff
@@ -0,0 +1,13 @@
+# Voltage sensor driver
+#
+# Copyright (c) 2023 FTP Technologies
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config VOLTAGE
+	bool "Voltage sensor driver"
+	default y
+	depends on DT_HAS_VOLTAGE_DIVIDER_ENABLED
+	select ADC
```

Comment:
This option should be available to user when an ADC is available.

```diff
-	select ADC
+	depends on ADC
```

Reply:
Thanks, fixed